### ISSUE TITLE
io.grpc.testing.integration.ProxyTest.smallLatency is flaky

### ISSUE THREAD

**Issue by ejona86** [grpc/grpc-java#2951]
This is a re-appearance of #2176, but that one is so old I feel bad with reopening it.

**Comment by ejona86** [comment]
Copied from #2176:
I saw this again at https://grpc-testing.appspot.com/job/gRPC-Java-PR-Windows/3128/console

io.grpc.testing.integration.ProxyTest > smallLatency FAILED
    java.lang.AssertionError: expected:<5.0E7> but was:<1.053576E8>
        at org.junit.Assert.fail(Assert.java:88)
        at org.junit.Assert.failNotEquals(Assert.java:743)
        at org.junit.Assert.assertEquals(Assert.java:519)
        at org.junit.Assert.assertEquals(Assert.java:609)
        at io.grpc.testing.integration.ProxyTest.smallLatency(ProxyTest.java:106)

**Comment by ejona86** [comment]
Seen at https://grpc-testing.appspot.com/job/gRPC-Java-PR-Windows/3294/console

```
io.grpc.testing.integration.ProxyTest > smallLatency FAILED
    java.lang.AssertionError: expected:<5.0E7> but was:<1.805384E8>
        at org.junit.Assert.fail(Assert.java:88)
        at org.junit.Assert.failNotEquals(Assert.java:743)
        at org.junit.Assert.assertEquals(Assert.java:519)
        at org.junit.Assert.assertEquals(Assert.java:609)
        at io.grpc.testing.integration.ProxyTest.smallLatency(ProxyTest.java:106)
```

**Comment by ejona86** [comment]
This time largeBandwidth. Seen at https://travis-ci.org/grpc/grpc-java/builds/345336422 (since rebuilt):
```
io.grpc.testing.integration.ProxyTest > largeBandwidth FAILED
    java.lang.AssertionError: expected:<1.048576E7> but was:<2279513.0>
        at org.junit.Assert.fail(Assert.java:88)
        at org.junit.Assert.failNotEquals(Assert.java:834)
        at org.junit.Assert.assertEquals(Assert.java:553)
        at org.junit.Assert.assertEquals(Assert.java:683)
        at io.grpc.testing.integration.ProxyTest.largeBandwidth(ProxyTest.java:167)
```

**Comment by creamsoup** [comment]
https://source.cloud.google.com/results/invocations/0663c79b-db78-4e41-af06-bf8892ea744f/targets/ (linux)

```
io.grpc.testing.integration.ProxyTest > largeBandwidth FAILED
    java.lang.AssertionError: expected:<1.048576E7> but was:<2.0628449291259527E7>
        at org.junit.Assert.fail(Assert.java:88)
        at org.junit.Assert.failNotEquals(Assert.java:834)
        at org.junit.Assert.assertEquals(Assert.java:553)
        at org.junit.Assert.assertEquals(Assert.java:683)
        at io.grpc.testing.integration.ProxyTest.largeBandwidth(ProxyTest.java:184)
```

**Comment by shivaspeaks** [comment]
I've tested with blaze test 10000 times and I see ~4 failures(truly flaky) for largeBandwidth().
smallLatency() test is not at all failing with 50ms, but it is failing with 10ms (~7/100).

Making latency 50ms was a short term fix (#2179 )

So, I'm doubting if enabling smallLatency with 50ms is an option here?
cc: @ejona86 @kannanjgithub
